k8s/apps/daemonsets.go

```diff
@@ -29,6 +30,8 @@ type DaemonSetOps interface {
 	UpdateDaemonSet(*appsv1.DaemonSet) (*appsv1.DaemonSet, error)
 	// DeleteDaemonSet deletes the given daemonset
 	DeleteDaemonSet(name, namespace string) error
+	// ValidateTerminatedDaemonSet validates if given daemonset is terminated
+	ValidateTerminatedDaemonSet(name, namespace string, timeout time.Duration) error
```

Comment:
nit: probably `ValidateDaemonSetTerminated`?

Reply:
I guess that does sound appropriate. I am following the same naming pattern as all other *Terminated methods.
I am torn between whether I should make an exception for this or just change in all the places.